SayTheirNames/Source/Utils/Image/UIImage+Resource.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  UIIcons+Resource.swift
+//  SayTheirNames
+//
+//  Created by Visal Rajapakse on 6/5/20.
+//  Copyright © 2020 Franck-Stephane Ndame Mpouli. All rights reserved.
+//
+
+import UIKit
+
+//collection of icons used
+extension UIImage{
+    enum STN {
```

Comment:
Using `#imageLiteral` will cause the app to crash if the asset is removed or rename. I thin kit would be best to exclude this.

Reply:
Another PR will treat those as warnings when used: #80

Reply:
@evilpenguin What would you suggest? Since force unwrapping will display a `Force Unwrapping Violation` warning.

Reply:
Maybe we have empty images `UIImage()` or maybe we decide on a single placeholder image design that never changes so our `UIImageView`s will always display something?

Reply:
Single placeholder seems legit.

Reply:
But I also think that giving a `UIImageView` a `nil` image is acceptable.